### PR TITLE
Suggest some simple samples

### PR DESCRIPTION
**Pull request by houdini91**

### CODE REVIEW

File: v1/evidence/artifact-evidence-exists.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name: artifact-evidence-exists
+description: "A rule to verify that the artifact produced any evidence"
+labels:
+  - sample-policy-bundle
+initiatives:
+  - client-policy
```
Comment (cppvik) [inline v1/evidence/artifact-evidence-exists.yaml:6]: It seems we cannot have an empty `format-type`, so we need to set it by default (I set it to `cyclonedx-json` in `v1/sboms/evidence-exists.yaml`).

File: v1/sboms/artifact-sbom-exists.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: artifact-evidence-exists
```
Comment (cppvik) [inline v1/sboms/artifact-sbom-exists.yaml:1]: Duplicate of `v1/sboms/evidence-exists.yaml`

File: v1/sboms/artifact-unsigned.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+name: artifact-unsigned
```
Comment (cppvik) [inline v1/sboms/artifact-unsigned.yaml:1]: So it's basically the same as `v1/sboms/evidence-exists.yaml`

**Comment by houdini91** [comment]
Ok updated the PR take another look

**Comment by cppvik** [comment]
We already have all of this in `dev`. Changes in text descriptions can be commited to the `wording-update` branch.

**Pull request closed without merging**